src/test/java/com/google/sps/servlets/AggregationServletTest.java

```diff
@@ -33,6 +37,13 @@
 @RunWith(JUnit4.class)
 public final class AggregationServletTest {
 
+  private final String LOGIN_URL = "/login";
+  private final String HOME_URL = "/index.html";
+  private final String AUTHORIZE_URL = "/authorize";
+  private final String TEST_USER_ID = "testUserId";
+  private final String TEST_USER_EMAIL = "testEmail";
+  private final String TEST_USER_AUTH_DOMAIN = "testAuthDomain";
```

Comment:
Optional: please consider adding a test util which holds the constants like TEST_USER_ID and util functions for mocking user services etc, so that they can be shared in all the test files.

Reply:
I will do this test refactoring in the next PR, update